packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button-matrix.stories.ts

```diff
@@ -0,0 +1,66 @@
+import type { StoryFn, Meta } from '@storybook/html';
```

Comment:
Do you think it would be useful to add this column to Table Keyboard Focus Theme Matrix story? I'm guessing yes since the column has focusable content so it'd be good to validate how its focus state renders. Though we don't currently include anchor columns or action menus in that test so I might be misinterpreting its purpose. @msmithNI feel free to offer an opinion (I know he's OOO today so we can proceed with our best guess).

https://60e89457a987cf003efc0a5b-qjyaqfanzg.chromatic.com/?path=/story/tests-table--table-keyboard-focus-theme-matrix

Reply:
I'd think this column should have it's own focus theme matrix test instead of putting all columns in a single test.

Reply:
Also probably reasonable to address in the style update follow-on

Reply:
I don't think it's necessarily needed because the table doesn't add focus visuals to the controls inside cell views, the controls themselves provide that focus styling. Per-column matrix tests could still make sense though I guess.

Reply:
I'll add a note to #2247 that we should also consider adding a focus theme matrix for the column.